tests/Unit/classes/PrestaShopAutoloadTest.php

```diff
@@ -48,6 +48,26 @@ public function testGenerateIndex()
         $this->assertEquals($data['OrderControllerCore']['path'], 'controllers/front/OrderController.php');
     }
 
+    public function testGenerateIndexWithoutOverride()
```

Comment:
If you can, please provide a description of each test in the form of "Given-When-Then".

This is important because tests are the warrant of the functional compliance of the system, they describe the way the system is supposed to work. And humans are generally better at understanding sentences rather than code 😄

Example:
```
/**
 * Given an invalid date string
 * When a DateTime object is instantiated with that string
 * Then an exception is raised
 * 
 * @expectedException \Exception
 *
public function testExceptionRaisedWhenCreatingFromInvalidDate()
{
    $dateString = 'invalid';
    new DateTime($dateString);
}
```

See this Wikipedia article for more information: [Acceptance criteria and tests](https://en.wikipedia.org/wiki/Acceptance_test%E2%80%93driven_development#Acceptance_criteria_and_tests)